sql3/test/defs/defs.go

```diff
@@ -210,6 +212,16 @@ func knownTimestamp() time.Time {
 	return tm
 }
 
+func knownSubSecondTimestamp() time.Time {
+	tm := knownTimestamp()
+	duration, err := time.ParseDuration("100100100ns")
```

Comment:
It might be a good idea to make this something like `duration, err := time.ParseDuration("100200300ns")` so that it's easier to be sure the right chunk of it is getting picked up by the ms/us/ns checks?

Reply:
Changed the sub-second values as suggested.